linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/grpc/GrpcClassifierConfig.scala

```diff
@@ -0,0 +1,72 @@
+package io.buoyant.linkerd.protocol.h2.grpc
+
+import com.fasterxml.jackson.annotation.JsonSubTypes
+import com.fasterxml.jackson.annotation.JsonSubTypes.Type
+import io.buoyant.linkerd.protocol.h2._
+
+@JsonSubTypes(Array(
+  new Type(
+    value = classOf[NeverRetryableConfig],
+    name = "io.l5d.h2.grpc.neverRetryable"
+  ),
+  new Type(
+    value = classOf[AlwaysRetryableConfig],
+    name = "io.l5d.h2.grpc.alwaysRetryable"
+  ),
+  new Type(
+    value = classOf[DefaultConfig],
+    name = "io.l5d.h2.grpc.default"
+  ),
+  new Type(
+    value = classOf[RetryableStatusCodesConfig],
+    name = "io.l5d.h2.grpc.retryableStatusCodes"
+  )
+))
+abstract class GrpcClassifierConfig extends H2ClassifierConfig {
```

Comment:
I mean, it's about three lines of boilerplate, right? Or is there something I'm missing?